lib/build.js

```diff
@@ -60,6 +60,7 @@ module.exports = {
     this.stdout = new Buffer(0);
     this.stderr = new Buffer(0);
     atom.commands.add('atom-workspace', 'build:trigger', this.build.bind(this));
+    atom.commands.add('atom-workspace', 'build:toggle-view', this.toggleView.bind(this));
```

Comment:
Maybe this command should be added in `build-view.js` instead since it's purely a presentation thing?

This would also keep us from having to implement the simple pass-through function `toggleView` in this class.